src/app/monitoring/containers/live-logs/live-logs.component.scss

```diff
@@ -1,15 +1,43 @@
+@import '../../../core/core';
+
+$text-color: $gray-700;
+$text-color-light: $gray-600;
+
 .container {
-    margin-top: 1%;
+    margin-top: 0.5vh;
     max-width: 100vw;
 }
 .nav-container {
-    display: grid;
-    grid-template-columns: 13% 20% 11%;
-    grid-gap: 1em;
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: flex-start;
+    margin-bottom: 2vh;
+    align-items: center;
 }
-label {
-    align-self: end;
+.appId{
+    justify-self: stretch;
```

Comment:
Das ist keine Flexbox sondern eine Grid Property im Layout hat sie bei meinem Test auch keinen Effekt. Wenn du damit sagen willst, dass sich das Element ausbreiten soll, musst du flex-grow und flex-shrink benutzen. Flex-Grow hat aber sowieso schon den default Wert 1.

Reply:
okay, das habe ich bei der Umstellung von grid auf flex vergessen rauszunehmen, weil ich es in dem moment nicht als grid Property erkannt habe, ich nehme es raus.